test/common/testinfra.py

```diff
@@ -608,6 +609,27 @@ def sort_key(entry):
     # The details can't contain newline characters and should be brief
 
     def update_known_issue(self, number, err, details, context, timestamp=None):
+        def redact_audit_variables(message):
```

Comment:
Only used there, but I don't care either way: I moved it to the top level of testinfra.py, but not into the list of exported functions.